mlflow/utils/autologging_utils/versioning.py

```diff
@@ -62,7 +63,16 @@ def is_flavor_supported_for_associated_package_versions(flavor_name):
         associated packages.
     """
     module_name = FLAVOR_TO_MODULE_NAME[flavor_name]
-    actual_version = importlib.import_module(module_name).__version__
+
+    try:
+        actual_version = importlib.import_module(module_name).__version__
+    except AttributeError:
+        try:
+            actual_version = importlib.metadata.version(module_name)
+        except importlib.metadata.PackageNotFoundError:
+            # Some package do not publish version info in a standard format.
```

Comment:
```suggestion
            # Some packages (e.g. xxx) do not publish version info in a standard format.
```

can you add an example?